src/components/pages/Home.vue

```diff
@@ -36,21 +36,32 @@
           this.garments = response.data;
         })
         .catch(error => this.showError(error.message));
-      const vm = this;
-      fetch('https://raw.githubusercontent.com/ecvdbdx1617/github-for-fashion/master/content/cover.json')
-      .then(
-        (response) => {
-          if (response.status !== 200) {
-            console.log(response.status);
-            return;
+      const gh = new GitHub({
+        token: LoginStore.state.token,
+      });
+
+      const remoteRepo = gh.getRepo('ecvdbdx1617', 'github-for-fashion');
+      // const vm = this;         this will be usefull later
+      remoteRepo.getContents('master', 'content/cover.json', false).then((file) => {
+        let myJson = decodeURIComponent(escape(window.atob(file.data.content)));
+        myJson = JSON.parse(myJson);
+        Object.keys(myJson).forEach((key) => {
```

Comment:
By design, you already know the object has 2 keys and their names. No need to iterate over "all" keys